homeassistant/components/ezviz/switch.py

```diff
@@ -55,7 +55,7 @@ def __init__(
     @property
     def is_on(self) -> bool:
         """Return the state of the switch."""
-        return self.data["switches"][self._name]
+        return self.data["switches"].get(self._name,False)
```

Comment:
Wouldn't this mean the switch is unavailable instead?

Reply:
It means that the data from the ezviz server is not complete.

Reply:
I got that from the PR, the question was: Wouldn't that make the switch unavailable?

Reply:
Yes the switch may be unavailable. It only avoids the exception.